packages/manager/modules/pci/package.json

```diff
@@ -18,6 +18,7 @@
     "fast-sha256": "^1.1.0",
     "file-saver": "^2.0.1",
     "flag-icon-css": "~0.8.5",
+    "highlight.js": "^11.6.0",
```

Comment:
Hi @cbourgois, @frenautvh,
Are we allowed to use this library?
Thanks!

Reply:
Hi @anooparveti , the library is quite large in size as we can see here : https://bundlephobia.com/package/highlight.js@11.7.0
I would suggest it's ok to use if we ensure that the section that is importing it is lazyloaded (with oclazyload), so it only will be loaded when needed